pyiron/project.py

```diff
@@ -165,16 +168,7 @@ def create_job(self, job_type, job_name, delete_existing_job=False):
         Returns:
             GenericJob: job object depending on the job_type selected
         """
-        job = JobType(
-            job_type,
-            project=ProjectHDFio(project=self.copy(), file_name=job_name),
-            job_name=job_name,
-            job_class_dict=self.job_type.job_class_dict,
-            delete_existing_job=delete_existing_job,
-        )
-        if self.user is not None:
-            job.user = self.user
-        return job
+        return super().create_job(job_type=job_type, job_name=job_name, delete_existing_job=delete_existing_job)
```

Comment:
Does this work? I agree it should work once it is mapped to `pr.create` but currently I have the feeling this might break the functionality so I would prefer to move it to a separate pull request.

Reply:
You bet, why wouldn't it? The code is line for line the exact same, and `pyiron.project` is anyhow just importing `JobClass` from pyiron_base.

Sure, I have no problem moving to another PR. There's some related changes I want to make anyhow now that we've got all this creator stuff into base.